### DISCUSSION TITLE
Adicionar dependência sem release

### DISCUSSION THREAD

**Discussion by gamboajorge49** [HashLoad/boss#78] [Q&A] [answered]
Olá, estou tentando adicionar uma dependência para um projeto que estou fazendo, porém ao executar o seguinte comando, `boss install github.com/gamboajorge49/logger4D ` , o boss traz a seguinte saída.

> [INFO ] Installing modules in project path
[INFO ] Building cache files...
[INFO ] Updating cache of dependency logger4d
[INFO ] Installing modules
[INFO ] Processing dependency logger4d
[WARN ] Tag not found , using semantic now...
[WARN ]   No candidate to version for >0.0.0. Using master branch
[ERROR]   Error on switch to needed version from dependency github.com/gamboajorge49/logger4d reference not found

Pela minha interpretação é por que ainda não criei nenhuma release e o nome da minha branch é main, ao invés de master.
É isso mesmo ou estou fazendo algo de errado ao adicionar a dependência ou no repositório do meu projeto ?

**Comment by viniciussanchez** [comment, marked as answer]
O Boss usa o semantic version pra controlar, então se não ter uma release liberada seria ruim pra ele. Mas essa questão da branch master / main preciso verificar. Desconhecia esse processo. Vou verificar

**Comment by arvanus** [comment]
Só comentando, realmente a tendência é mudar a branch principal de `master` para `main`

**Comment by LucasSouzaAndrade** [comment]
Olá, estou com essa mesma situação no meu ambiente
 `boss i https://github.com/pootmhicnought/Delphi_AMQP.git`
![image](https://github.com/HashLoad/boss/assets/35232041/088db30a-696c-4228-8e3a-05a9a4a7e6a0)

Projeto
![image](https://github.com/HashLoad/boss/assets/35232041/6827bf52-8f2f-4dfc-a818-c52978bcb20b)
 Existe alguma solução?

**Reply by gamboajorge49** [reply]
No meu caso como era um repositório público, e eu não poderia esperar, eu achei melhor fazer um fork do repositório e adicionar a release, para assim poder utilizar no meu projeto.

**Reply by LucasSouzaAndrade** [reply]
Vou tentar isso
